<?xml version="1.0"?>
<robot name="lidar_sim" xmlns:xacro="http://ros.org/wiki/xacro">

  <gazebo reference="base_link">
    <gravity>0</gravity>
  </gazebo>

  <gazebo>
    <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
    </plugin>
    <plugin name="tc_sim_plugin" filename="libtc_sim_plugin.so">
    </plugin>
    <!-- <plugin name="p3d_base_controller" filename="libgazebo_ros_p3d.so"> -->
    <!-- <bodyName>base_link</bodyName> -->
    <!-- <frameName>world</frameName>
    <topicName>odom</topicName>
    <updateRate>30.0</updateRate> -->
    
<!-- </plugin> -->
  </gazebo>

</robot>
